test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,25 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
+        assertIsNotNull();
+        assertIsNotNull("abc");
+        assertIsNotNull("");
+        assertIsNotNull("abc123", "abcdef", "ghi");
+
+        assertIsNull((Object) null);
+        assertIsNull( null, null);
+    }
+
+    private void assertIsNull(Object... object) {
+        assertTrue(Utils.isAnyNull(object));
+    }
+
+    private void assertIsNotNull(Object... object) {
+        assertFalse(Utils.isAnyNull(object));
```

Comment:
Do note that it is not necessary to abstract the `assertTrue` and `assertFalse` as separate methods (it was done previously to avoid having to type out Arrays.asList each time). However, it is also considered good practice to follow existing coding styles.